dbms/src/Flash/Mpp/MPPTunnel.cpp

```diff
@@ -25,6 +26,8 @@ namespace DB
 namespace FailPoints
 {
 extern const char exception_during_mpp_close_tunnel[];
+extern const char random_tunnel_write_failure_failpoint[];
```

Comment:
The above two failpoints are not used in this file?

Reply:
Remove useless failpoint declaration.